config/test.exs

```diff
@@ -1,6 +1,6 @@
 use Mix.Config
 
 config :capture, Capture.Repo,
-  database: "capture_test",
+  database: "capture_dev",
```

Comment:
This should be left as `capture_test`, because this file is the configuration file for the test environment.

Reply:
Hey Ryan, is there an ecto way to create the `capture_test` database and insert the table into it?

We created a `capture_test` database, and created the responses table, but the table keeps going to `capture_dev`